packages/cosmic-swingset/lib/ag-solo/vats/vat-http.js

```diff
@@ -37,13 +37,18 @@ function build(E, D) {
   });
   harden(readyForClient);
 
-  let handler = {};
-  const registeredHandlers = [];
+  const handler = {};
+  const registeredURLHandlers = new Map();
```

Comment:
A registration Map, rather than a list, makes a lot of sense. Do we still want to allow multiple registered functions per URL, or would it be cleaner to allow only one per URL?

Reply:
I have a plan to break out the handlers into objects with properties identifying what `msg.type` field they apply to.  At that point, we can keep one handler per URL, which will be an object keyed by the `msg.type` pointing to its latest installed implementation.

Reply:
(For compatibility right now we need multiple functions.)